hugo-config/docs.toml

```diff
@@ -78,6 +78,18 @@ category = "categories"
   identifier = "/kb"
   weight     = 600
 
+[[menu.main]]
```

Comment:
These are showing up as top level navigation which we don't want, its not bad to list all developer resources and supporting your customer articles but they should be local navigation after you select "Knowledge Base" just like the nav shows up under "Packaging an Application" or the other top level nav options.